analysis/analysisrunner.cpp

```diff
@@ -227,11 +240,11 @@ QString AnalysisRunner::humanFileSize() const
 {
     int unit;
     const char *units [] = {" Bytes", " kB", " MB", " GB"};
-    quint64 size = fileSize(); // or whatever
+    qint64 size = fileSize(); // or whatever
```

Comment:
Windows is using signed filesize too.

https://msdn.microsoft.com/en-us/library/windows/desktop/aa383713(v=vs.85).aspx

QuadPart
A signed 64-bit integer.